src/core/usecases/userConfigs.ts

```diff
@@ -37,6 +37,7 @@ export type UserConfigs = Id<
         doDisplayMySecretsUseInServiceDialog: boolean;
         bookmarkedServiceConfigurationStr: string | null;
         selectedProjectId: string | null;
+        accessToken: string | null;
```

Comment:
We shouldn't store the accessToken but rather ask for a new token with the keycloakParams if any.  
Anyway se should never store the accessToken in Vault or in local storage.